geps/gep-1494/index.md

```diff
@@ -0,0 +1,144 @@
+
+* A way for Chihiro the Cluster Admin to configure a default Authentication and/or Authorization config for some set of HTTPRoutes.
+
+* Optionally, a way for Ana to have the ability to disable Authentication and/or Authorization for specific routes when needed, allowing certain routes to not be protected.
+
+
+## Non-Goals
+
+* Handling all possible authentication and authorization schemes. Handling a (preferably large) subset of authentication and authorization is acceptable.
+
+
+## Deferred Goals
+
+* (Not performed during the Provisional Phase) Defining the API or configuration to be used.
+
+## Introduction
+
+A common request for Gateway API has been a way to direct implementations to automatically request various forms of authentication (AuthN) and authorization (AuthZ). (see the [GEP Issue](https://github.com/kubernetes-sigs/gateway-api/issues/1494) for some of the history.)
+
+**Authentication (AuthN for short)** refers to proving the requesting party's identity in some fashion.
+
+**Authorization (AuthZ for short)** refers to making decisions about what a party can access _based on their Authenticated identity_.
+
+In this document, we'll use Auth* to represent "AuthN and maybe AuthZ as well".
+
+This capability is useful for both Ana (who wants to have something for AuthN and AuthZ that ensures security without it needing to be built into her app), and Chihiro (who wants to be able to ensure that the platform as a whole conforms to some given level of security).
+
+### Common authentication methods
+
+Before discussing any proposed Auth* solution, it's important to discuss some AuthN methods that are often used in securing modern applications.
+
+#### Basic HTTP Auth
+
+In Basic HTTP Auth, a server asks for authentication as part of returning a `401` status response, and the client includes an `Authorization` header that includes a Base64-encoded username and password.
+
+Because the password is only _encoded_ and not _encrypted_, Basic Auth is totally unsafe when used outside of an encrypted session (like a HTTPS connection).
+
+Basic auth is defined in [RFC-7617](https://datatracker.ietf.org/doc/html/rfc7617).
+
+#### TLS Client Certificate Authentication
+
+TLS includes the possibility of having both the client and server present certificates for the other party to validate. (This is often called "mutual TLS", but is distinct from the use of that term in Service Mesh contexts, where it means something more like "mutual TLS with short-lifetime, automatically created and managed dynamic keypairs for both client and server").
+
+In this case, the server authenticates the client based on the client presenting a certificate that's signed by an authority that's also trusted by the server's trust chain. Some implementations also allow details about the certificate to be passed through to backend clients, to be used in authorization decisions.
```

Comment:
I think the "also" is misplaced, since it's about the server also verifying a client, not about the client's cert being trusted by the same CA as ??.
The "trust chain" seems weird as well. Generally we have trust roots on a server, and the chain is presented by the client.
Since the client side authentication process of the server is implied prior knowledge, I think this can be condensed a bit.

I suggest:

`In this case, the server also authenticates the client, based on the certificate chain presented by the client. Some implementations also allow details about the certificate to be passed through to backend clients, to be used in authorization decisions`